pkg/controller/master/setup.go

```diff
@@ -11,6 +12,7 @@ import (
 func Setup(ctx context.Context, server *server.Server) error {
 	scaled := config.ScaledWithContext(ctx)
 	go leader.RunOrDie(ctx, "", "harvester-controllers", server.K8s, func(ctx context.Context) {
+		jwe.NewJWETokenManager(scaled.CoreFactory.Core().V1().Secret())
```

Comment:
prefer to add link break here since the code below are not in the same code block.